packages/next/src/next-devtools/dev-overlay/components/overview/segment-explorer.tsx

```diff
@@ -83,6 +91,20 @@ function PageSegmentTreeLayerPresentation({
     folderChildrenKeys.push(childKey)
   }
 
+  for (const fileChildSegment of filesChildrenKeys) {
+    const childNode = node.children[fileChildSegment]
+    if (!childNode || !childNode.value) continue
+
+    // If it's a page node, we can use it as the page child
+    if (
+      childNode.value.type !== 'layout' &&
+      childNode.value.type !== 'template'
+    ) {
+      pageChild = childNode
+      break // We only need one page child
```

Comment:
The simulation should be available on each level so that you can simulate all possible states. page-like segments aren't the only segments that can error or suspend. That can happen anywhere. Most notably simulation on layouts is absent.